modules/ocf_xmpp/manifests/init.pp

```diff
@@ -0,0 +1,57 @@
+class ocf_xmpp {
+  # Needed to connect to MySQL
+  package { ['lua-dbi-mysql']: }
+
+  user { 'prosody':
```

Comment:
use ocfprosody to not conflict with usernames?

Reply:
The `prosody` name has been reserved for a while, and this is the default name used by the package anyways.